src/Android/Android.csproj

```diff
@@ -61,7 +61,7 @@
     <JavaMaximumHeapSize>1G</JavaMaximumHeapSize>
     <AndroidUseSharedRuntime>false</AndroidUseSharedRuntime>
     <EmbedAssembliesIntoApk>true</EmbedAssembliesIntoApk>
-    <AndroidLinkMode>Full</AndroidLinkMode>
```

Comment:
So linker is no longer necessary?

Reply:
Full linking isn't supported with AOT, only "assemblies only" (which is what happens when this line is removed outright)